### PR TITLE
Greedy Algorithm-2

### PR DESCRIPTION
**Pull request by mansijain980**
# Have you read the [Contributing Guidelines](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/blob/master/Contributing.md)
- [X] YES
- [ ] NO

## Project
Added  Kruskal's Minimum Tree Algorithm using Union

## What is the current behavior?
- Issue Number: #170 

## Category
- [ ] 𝑨𝑺𝑲 𝒀𝒐𝒖𝒓 𝑫𝒐𝒖𝒃𝒕𝒔
- [ ] 𝑴𝒊𝒔𝒄𝒆𝒍𝒍𝒂𝒏𝒆𝒐𝒖𝒔 𝑺𝒕𝒖𝒇𝒇.
- [x] 𝑫𝑺𝑨
- [x] 𝑪𝒐𝒎𝒑𝒆𝒕𝒊𝒕𝒊𝒗𝒆 𝑷𝒓𝒐𝒈𝒓𝒂𝒎𝒎𝒊𝒏𝒈
- [ ] 𝑻𝒆𝒄𝒉𝒏𝒊𝒄𝒂𝒍 𝑺𝒖𝒃𝒋𝒆𝒄𝒕
- [ ] 𝑳𝒐𝒘 𝑳𝒆𝒗𝒆𝒍 𝑫𝒆𝒔𝒊𝒈𝒏
- [ ] 𝑷𝒓𝒐𝒋𝒆𝒄𝒕𝒔
- [ ] 𝑰𝒎𝒑𝒐𝒓𝒕𝒂𝒏𝒕 𝑩𝒐𝒐𝒌𝒔 𝒂𝒏𝒅 𝑹𝒆𝒔𝒐𝒖𝒓𝒄𝒆𝒔
- [ ] 𝑩𝒆𝒉𝒂𝒗𝒊𝒐𝒖𝒓𝒂𝒍 𝑰𝒏𝒕𝒆𝒓𝒗𝒊𝒆𝒘 𝑸𝒖𝒆𝒔𝒕𝒊𝒐𝒏s

## Checklist
- [x] I've read the [Contributing Guidelines](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/blob/master/CONTRIBUTING.md).
- [x] I've checked the issue list before deciding what to submit.
- [x] I've follow [Tree of Index](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation#tree-of-index).
- [x] I've add work in proper folder
- [x] I've not delete/modify previous codes.

### CODE REVIEW

**Comment by mansijain980** [comment]
@AkashSingh3031 Please see to it.

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
Add input function also
Add image of `Take input and output of code` in comment

**Comment by mansijain980** [comment]
Sure

**Comment by mansijain980** [comment]
image
image

**Comment by mansijain980** [comment]
image

File: 1]. DSA/2]. Algorithms/06]. Greedy/Python/Kruskal’s Minimum Spanning Tree Algorithm.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Python program for Kruskal's algorithm to find
+# Minimum Spanning tree
+# undirected and weighted graph
+# Kruskal's algorithm finds a minimum spanning forest of an undirected edge-weighted graph. 
+# If the graph is connected, it finds a minimum spanning tree. 
+# It is a greedy algorithm in graph theory as in each step it adds the next lowest-weight edge 
+ 
+
+# class 
+class DisjointSet:
```
Comment (AkashSingh3031) [inline 1]. DSA/2]. Algorithms/06]. Greedy/Python/Kruskal’s Minimum Spanning Tree Algorithm.py:10]: Add Screenshot of `Output of Code`

**Review by AkashSingh3031 — COMMENTED** [review]
Accepted

**Comment by mansijain980** [comment]
As I have a keen interest in DSA, Can I push more algos?

**Comment by AkashSingh3031** [comment]
> As I have a keen interest in DSA, Can I push more algos?

Yes, feel free to contribute

**Pull request merged**